### PR TITLE
[mod_stats]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_stats
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_stats/mod_stats.xml [hunk 1]
```diff
@@ -28,78 +28,83 @@
 					class="btn-group btn-group-yesno"
 					default="0"
 					label="MOD_STATS_FIELD_SERVERINFO_LABEL"
```
Comment (andrepereiradasilva) [inline modules/mod_stats/mod_stats.xml]: shouldn' default and class be moved down?

File: modules/mod_stats/mod_stats.xml [hunk 2]
```diff
@@ -28,78 +28,83 @@
...
+					default="900"
+				/>
+
 				<field
 					name="cachemode"
 					type="hidden"
 					default="static">
-					<option
-						value="static"></option>
+					<option value="static"></option>
```
Comment (andrepereiradasilva) [inline modules/mod_stats/mod_stats.xml]: shouldn't the > be in a new line?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on eb4638ed03835fc5cba165a3227f890c62abc9df<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10592">issues.joomla.org/joomla-cms/10592</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on eb4638ed03835fc5cba165a3227f890c62abc9df<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10592">issues.joomla.org/joomla-cms/10592</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10592">issues.joomla.org/joomla-cms/10592</a>.</sub>

**Pull request closed without merging**